src/test/java/microsoft/exchange/webservices/data/util/TimeZoneUtilsTest.java

```diff
@@ -49,12 +49,15 @@ public void testGetMicrosoftTimeZoneNameBad() {
     // null-argument is not allowed.
     try {
       Assert.fail(TimeZoneUtils.getMicrosoftTimeZoneName(null));
-    } catch (final IllegalArgumentException ignored) {}
-
-    // Case-insensitive ID is not supported.
-    checkGetMicrosoftTimeZoneName("africa/abidjan", "UTC");
+    } catch (final IllegalArgumentException ignored) {
+    }
   }
 
+  @Test
+  public void testCaseInsensitiveIdNotSupported() {
+    // Case-insensitive ID is not supported so the map look up returns null.
+    checkGetMicrosoftTimeZoneName("africa/abidjan", null);
```

Comment:
I'll keep trying to contribute. :+1: 

I just have to get a senior director to sign the CLA, and the other PR should be unblocked.